lang/cs.json

```diff
@@ -1,26 +1,85 @@
 {
+  "Audio Player": "Audio Přehravač",
+  "Video Player": "Video Přehravač",
   "Play": "Přehrát",
   "Pause": "Pauza",
+  "Replay": "Spustit znovu",
   "Current Time": "Aktuální čas",
   "Duration": "Doba trvání",
   "Remaining Time": "Zbývající čas",
-  "Stream Type": "Stream Type",
+  "Stream Type": "Typ streamu",
   "LIVE": "ŽIVĚ",
   "Loaded": "Načteno",
   "Progress": "Stav",
+  "Progress Bar": "Postupní pruh",
+  "progress bar timing: currentTime={1} duration={2}": "{1} z {2}",
   "Fullscreen": "Celá obrazovka",
   "Non-Fullscreen": "Zmenšená obrazovka",
   "Mute": "Ztlumit zvuk",
   "Unmute": "Přehrát zvuk",
   "Playback Rate": "Rychlost přehrávání",
   "Subtitles": "Titulky",
-  "subtitles off": "Titulky vypnuty",
+  "subtitles off": "Bez titulků",
   "Captions": "Popisky",
```

Comment:
Using Google Translate for "Subtitles for the Deaf" or "Subtitles for the Hard of Hearing", I get "Titulky pro neslyšící", whereas "Popisky" gets translated back as "labels".